### PR TITLE
Create Flat button grid component

### PR DESCRIPTION
**Pull request by tanweer919**

### CODE REVIEW

File: src/components/FlatButtonGrid/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import classes from "./index.module.scss";
+export interface FlatButtonGridProps {
+  items: String[];
+}
+const FlatButtonGrid = (props: FlatButtonGridProps) => {
+  return (
+    <div className={classes.flatButtonGrid}>
```
Comment (silvanojulio) [inline src/components/FlatButtonGrid/index.tsx:8]: please, use classes.root instead of classes.flatButtonGrid.

**Review by silvanojulio — CHANGES_REQUESTED** [review]
In case the component does not require more functionality (icons, setting the background color, etc...) this is OK.

File: src/components/FlatButtonGrid/index.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import classes from "./index.module.scss";
+export interface FlatButtonGridProps {
+  items: String[];
```
Comment (silvanojulio) [inline src/components/FlatButtonGrid/index.tsx:4]: The component should receive classNames to merge it with the root style

File: src/components/FlatButtonGrid/index.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+}
+const FlatButtonGrid = (props: FlatButtonGridProps) => {
+  return (
+    <div className={classes.flatButtonGrid}>
+      {props.items.map((item) => (
+        <div className={classes.gridItem}>{item}</div>
```
[the remark this answers is not in the document]
Comment (tanweer919) [reply src/components/FlatButtonGrid/index.tsx]: done

File: src/components/FlatButtonGrid/index.tsx (hunk 1 above)
Comment (tanweer919) [reply src/components/FlatButtonGrid/index.tsx]: done

File: src/components/FlatButtonGrid/index.tsx (hunk 2 above)
Comment (tanweer919) [reply src/components/FlatButtonGrid/index.tsx]: done

**Review by silvanojulio — APPROVED** [review]

**Pull request merged**